src/api/app/controllers/webui/reports_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Webui::ReportsController < Webui::WebuiController
+  before_action :require_login
+  after_action :verify_authorized, except: [:create]
```

Comment:
Hm, why?

Reply:
This is not correct. Fallout from the mob session and didn't realize it was still skipped